package/libsolv.changes

```diff
@@ -1,3 +1,16 @@
+-------------------------------------------------------------------
+Thu Jan  4 11:02:29 UTC 2018 - tchvatal@suse.com
+
+- Sort a bit with spec-cleaner
+- Use python/ruby/etc condition names to match what other packages do
+  in order to make sure we are enabling/disabling stuff within prjcfg
+- Silence the source unpacking
+- As fedora/rh uses its own package make sure we use our guidelines
+  for cmake macro usage to get the most of the options for free
```

Comment:
I call shenanigans. You can do the right thing without being rude about it. This is a reference, multi-distro spec. I use it on OBS for testing across the different distros that I'm the maintainer of libsolv on for various things too.

Reply:
Lovely but it didn't build well anyway, and since this spec is foremost inherited for openSUSE/SLES it should by default do well that task and be multidistro later.